### PR TITLE
Issue #332: Update README.md to add checkNamesForContextStrategyByTokenOrAncestorSet property

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #332: Update README.md to add checkNamesForContextStrategyByTokenOrAncestorSet property

### CODE REVIEW

**Comment by rdiachenko** [comment]
@HuGanghui please rebase

**Comment by HuGanghui** [comment]
rebased.

**Review by rdiachenko — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -56,23 +56,38 @@ but also consider a wider context that new code introduces violations outside of
...
  | supportContextStrategyChecks                     | String has user defined Checks that support context strategy | [String](https://checkstyle.sourceforge.io/property_types.html#String) | null          |
- | neverSuppressedChecks                            | String has user defined TreeWalker Checks to never suppress if files are touched, split by comma. This property is useful for Checks that place violation on whole file or on not all (first/last) occurrence of cause/violated code. | [String](https://checkstyle.sourceforge.io/property_types.html#String) | null          |
  | checkNamesForContextStrategyByTokenOrParentSet | String has user defined TreeWalker Checks that need modify violation nodes to their parent node to expand the context scope, split by comma | [String](https://checkstyle.sourceforge.io/property_types.html#String) | null          |
+ | checkNamesForContextStrategyByTokenOrAncestorSet | String has user defined TreeWalker Checks that need modify violation nodes to their ancestor node to expand the context scope, split by comma | [String](https://checkstyle.sourceforge.io/property_types.html#String) | null          |
```
[the remark this answers is not in the document]
Comment (HuGanghui) [reply README.md:75]: ok, done

**Review by rdiachenko — APPROVED** [review]
LGTM

**Pull request merged**